src/popover/popover.jsx

```diff
@@ -118,7 +120,7 @@ const Popover = React.createClass({
       position: 'fixed',
       top: anchor.top,
       left: anchor.left,
-      zIndex: 20,
+      zIndex: 0,
```

Comment:
I see two options:
1. Set back the magic `20` and later use a constant zIndex
2. Add constant zIndex file now and use it here